src/ray/gcs/gcs_server/gcs_placement_group_scheduler.h

```diff
@@ -469,14 +469,14 @@ class GcsPlacementGroupScheduler : public GcsPlacementGroupSchedulerInterface {
                       &group_to_bundles) override;
 
  protected:
-  /// Send a bundle PREPARE request to a node. The PREPARE request will lock resources
+  /// Send bundles PREPARE requests to a node. The PREPARE requests will lock resources
   /// on a node until COMMIT or CANCEL requests are sent to a node.
```

Comment:
Can you say 
```
NOTE: All of given bundles will be prepared on the same node. It is guaranteed that all of bundles are atomically prepared on a given node. 
```

Also, `It is guaranteed that all of bundles are atomically prepared ` -> this is guaranteed in our code right?

Reply:
updated